src/lib/auth/profile.ts

```diff
@@ -35,6 +36,7 @@ export async function upsertProfile(authUser: User) {
     .values({
       id: authUser.id,
       name: getNameFromAuthUser(authUser),
+      role: !!authUser.email && ADMIN_EMAILS.includes(authUser.email) ? 'admin' : 'user',
```

Comment:
imo let's remove this and make admin-ness manual (via drizzle)

Reply:
@soorria wdym here 👁️ 👁️